cs/src/Connections/TunnelRelayTunnelHost.cs

```diff
@@ -458,12 +458,14 @@ private void OnSshChannelOpening(object? sender, SshChannelOpeningEventArgs e)
                         return;
                     }
 
-                    if (!RemoteForwarders.ContainsKey(new SessionPortKey(sessionId, (ushort)portForwardRequest.Port)))
```

Comment:
I think the other checks above are unnecessary also, since their primary purpose was to get the `sessionId` for this lookup.

Reply:
Done